include/nbl/builtin/glsl/ext/MitsubaLoader/material_compiler_compatibility_impl.glsl

```diff
@@ -39,6 +39,11 @@ layout(set = 0, binding = 6, std430) restrict readonly buffer PREFETCH_INSTR_BUF
     nbl_glsl_MC_prefetch_instr_t data[];
 } prefetch_instr_buf;
 
+layout(set = 0, binding = 7, std430) restrict readonly buffer EMITTER_BUF
+{
+    nbl_glsl_MC_emitter_t data[];
+} emitter_buf;
```

Comment:
I think a lot of stuff in Example 22 and the Mistuba loader expects the next "free slot" to be 7, can you show me where you solve that?

Reply:
ok I see its taken care of in Mitsuba Loadeer cpp